components/eamxx/tests/coupled/dynamics_physics/homme_mam4xx_pg2/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/homme_mam4xx_pg2_output.yaml
 
 # Set homme's test options, so that we can configure the namelist correctly
 # Discretization/algorithm settings
-set (HOMME_TEST_NE 2)
+set (HOMME_TEST_NE 4)
```

Comment:
Any reason not to use NE2? It may help with valgrind/memcheck builds. Having both NE2 and NE4 tests forces us to keep an up to date IC file for both resolutions...

Reply:
No reason--I think ne4 was just easier for me and Pete to get working, but now that it does work we can switch it back.

Reply:
We'll have to rework this test in the next PR anyway, so I'll switch it back to NE2 then.